src/coreComponents/unitTests/meshTests/testVTKImport.cpp

```diff
@@ -450,7 +450,8 @@ TEST( VTKImport, medley )
     }
   };
 
-  string const medleyVTK = testMeshDir + "/medley.vtk";
+
+  string const medleyVTK = testMeshDir + "/supportedElements.vtk";
 
   TestMeshImport( medleyVTK, validate );
```

Comment:
```suggestion

  string const medleyVTK = testMeshDir + "/supportedElements.vtk";
  string const medleyVTK42 = testMeshDir + "/supportedElements42.vtk";

  TestMeshImport( medleyVTK, validate );
  TestMeshImport( medleyVTK42, validate );
```

In this way we can remove all duplicated code (as done for the cube tests):

https://github.com/GEOSX/GEOSX/blob/2d1b5618783558dbe044790bbbbaa5e5f24d65ca/src/coreComponents/unitTests/meshTests/testVTKImport.cpp#L459-L737

Reply:
Tests are combined now.